<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Juego">
     <attribute name="title">
      <string>Juego</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-230</y>
        <width>771</width>
        <height>851</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/wallpaper.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/mastermind.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIniciar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eras Bold ITC</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 74, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>INICIAR JUEGO</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="botonVerde">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>250</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/ball_green.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>180</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/ball_blue.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonRojo">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>320</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/ball_red.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonAnaranjado">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>510</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/ball_orange.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonMorado">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>390</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/ball_purple.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonAmarillo">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>450</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/ball_yellow.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIniciar_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eras Bold ITC</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 74, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TERMINAR JUEGO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deshacer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eras Bold ITC</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 74, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DESHACER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CargarJuego">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eras Bold ITC</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 74, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CARGAR JUEGO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="guardarJuego">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eras Bold ITC</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 74, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>GUARDAR JUEGO</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdHoras">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="level">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>NIVEL: FÁCIL</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdMinutos">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>530</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdSegundos">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>530</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NombreJugador">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 145, 145);</string>
      </property>
      <property name="text">
       <string>Nombre de jugador...</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>625</x>
        <y>0</y>
        <width>41</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="bton1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>500</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bton2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>500</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bton3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>500</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bton4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>500</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="califica2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>500</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="califica4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>530</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="califica3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>530</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="califica1">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>500</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/images/circle.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="NumeroJugada">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>500</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calificarJugada">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eras Bold ITC</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 74, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CALIFICAR  JUGADA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Records">
     <attribute name="title">
      <string>Récords</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>701</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Forte</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Records mundiales de Master Mind</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Generar Pdf</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Ver Pdf</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_76">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/images/images/guinnes.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Ayuda">
     <attribute name="title">
      <string>Ayuda</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver Ayuda</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
